README.md

```diff
@@ -372,6 +372,9 @@ The name of the key pair associated with this instance. This must be an existing
 #####`instance_type`
 *Required* The type to use for the instance. This parameter is set at creation only; it is not affected by updates. See [Amazon EC2 Instances](http://aws.amazon.com/ec2/instance-types/) for available types.
 
+#####`tenancy`
+*Optional* Dedicated instances are Amazon EC2 instances that run in a virtual private cloud (VPC) on hardware that's dedicated to a single customer. Detaults to shared hardware.
```

Comment:
"Detaults" => "Defaults" ; maybe also mention how the param relates to dedicated instances?